__tests__/topics.test.js

```diff
@@ -35,3 +35,4 @@ describe("GET /api/topics", () => {
 });
 })
 
+//edit for jay
```

Comment:
Maybe here you could test for the data types of the properties too i.e. 
expect.objectContaining({ key: expect.any(data type)})